tests/testData/settingsFunctions.ts

```diff
@@ -16,27 +14,33 @@ export async function checkItemInSettingsMenu(
 }
 
 export async function checkSettingInSettingsMenu(
```

Comment:
Same comment as before regarding this, do not change files that you are not updating please